### ISSUE TITLE
[Ruby] Enable-API link in the README is incorrect

### ISSUE THREAD

**Issue by dazuma** [googleapis/gapic-generator#2195]
The link for enabling the API in the generated README is incorrect.

For example, currently for Redis, the generated README contains this line:

```md
3. [Enable the Google Cloud Memorystore for Redis API.](https://console.cloud.google.com/apis/api/redis)
```

This link does not work. A link that works correctly might be:

```md
3. [Enable the Google Cloud Memorystore for Redis API.](https://console.cloud.google.com/apis/library/redis.googleapis.com)
```

**Comment by dazuma** [comment]
This is still broken in 0.15.0. Now it's creating links of the form `https://console.cloud.google.com/apis/library/api/redis.googleapis.com`. This does not work.

Either of the following alternatives appear to work:
* https://console.cloud.google.com/apis/library/redis.googleapis.com
* https://console.cloud.google.com/apis/api/redis.googleapis.com

The former is probably preferred, as it results in fewer redirects.

**Comment by dazuma** [comment]
Fix verified in 0.15.1.

**Comment by jbolinger** [comment]
thanks for confirming @dazuma! Closing.
